apps/mobile/src/screens/CreateGroupFlow/CreateGroupParentGroups.js

```diff
@@ -37,43 +55,65 @@ export default function CreateGroupParentGroups ({ navigation }) {
     dispatch(updateGroupData({ parentIds }))
   }, [parentIds]))
 
+  const selectedGroups = parentGroupOptions.filter(group => isChosen(group))
+
   return (
-    <KeyboardFriendlyView style={styles.container}>
-      <Text style={styles.heading}>{t('Is this group a member of other groups?')}</Text>
-      <Text style={stepStyles.subHeading}>{t('Please select below:')}</Text>
-      {/* TODO: ItemSelectorModal  */}
-      {/* <FlashList
-        style={stepStyles.parentGroupListContainer}
-        data={parentGroupOptions}
-        renderItem={({ item }) => (
-          <ItemChooserItemRow item={item} chosen={isChosen(item)} toggleChosen={toggleChosen} />
+    <KeyboardFriendlyView className="bg-background p-5 flex-1">
+      <Text className="text-foreground text-xl font-bold pb-2.5">{t('Is this group a member of other groups?')}</Text>
+      <Text className="text-foreground/80 mb-2.5">{t('Please select below:')}</Text>
+
+      <View className="border border-foreground/20 bg-card rounded-lg p-4">
+        <TouchableOpacity 
+          className="flex-row justify-between items-center"
+          onPress={() => groupSelectorModalRef.current.show()}
+        >
+          <Text className="text-foreground/90 font-bold">{t('Parent Groups')}</Text>
+          <View className="border border-foreground rounded-full p-1">
+            <Icon className="text-foreground text-xl" name='Plus' />
+          </View>
+        </TouchableOpacity>
+        
+        <ItemSelectorModal
```

Comment:
Bravo. Love to see this component in use, and good job figuring it out. Haven't ran this yet, but am wondering if we just want an inline ItemSelector here instead? Will look, and if it works we don't need to do more now, but worth considering.